README.md

```diff
@@ -67,20 +64,60 @@ When building your own application, you will need to:
 - **Create a data store:** Log into the [Vectara Console](https://console.vectara.com/) and create a data store(https://docs.vectara.com/docs/console-ui/creating-a-corpus).
 - **Add data to the data store.** You can use [Vectara Ingest](https://github.com/vectara/vectara-ingest/blob/main/README.md#quickstart) to crawl datasets and websites, or use our [Indexing APIs](https://docs.vectara.com/docs/api-reference/indexing-apis/indexing) directly.
 
-### Running Your Custom App
+### Running Your Custom App 
 
-If you choose `[Create Your Own]` from the application selection prompt, you will be asked to provide:
+After you setup your datastore and index your data, you can run `vectara-answer` against this data. Run `bash docker/run.sh`, and choose `[Create Your Own]` from the application selection prompt, you will be asked to provide:
 
-- your Vectara customer ID
-- the ID of the corpus you created as a prerequisite to this process
-- the API key of your selected Vectara corpus (**NOTE: Depending on your set up, this may be visible to users. To ensure safe sharing, ensure that this key is set up to only have query access.**)
+- your Vectara `customer ID`
+- your `corpus ID`: the ID of the corpus you created as a prerequisite to this process
+- the `API key` of your selected Vectara corpus
 - any sample questions to display on the site, to get your users started.
 
-Once provided, the values above will go into your own customized configuration, and your site will be ready to go via `npm start` or the Docker script.
+Once provided, the values above will go into your own customized configuration, and your site will be ready to go via `npm start`.
+
+ **NOTE**: Depending on your set up and how you use vectara-answer, the api_key may be visible to users. To ensure safe sharing, ensure that this key is set up to only have query access.
+
+
+## Running with Docker
+
+The Quickstart is a great way to get started with your application within minutes. `vectara-answer` also supports an option that uses `Docker` and allows more customization to the application configuration. 
+
+### Setting up the environment
+
+First you will have to install a few more things:
+- If it's not already installed, install [Docker](https://docs.docker.com/engine/install/). Then make sure it's running on your machine.
+- Make sure you have a working version of Python>=3.8.
+- pyyaml: `pip3 install pyyaml`.
 
-### Deploying Your App
+### Configuring your application
 
-You can deploy `vectara-answer` on cloud platforms such as AWS, Azure, or GCP as well as on specialized cloud services like Render or Heroku.
+When running with Docker, vectara-answer requires a file called `secrets.toml` in the root directory. This file uses the TOML format, which supports the definition of one or more profiles. Under each profile you can add the line api_key="XXX" where XXX is the Vectara API key you want to use in that profile.
```

Comment:
Ah okay - I didn't see "LEGACY DEPLOYMENT"
So two things here:
- I would not call it legacy. I think I want to position these as just two ways to do it. Especially since we don't have cloud deployment for the Docker version and we ahve customers using it. So perhaps move my updates from README around Docker to a new file but we can call it "Docker deployment" or something like this
- I also find that configuaration description (at the bottom of README) makes sense in the context of config.yaml but is a bit more confusing when you think about the Quickstart and .env. So perhaps we need to clarify. Maybe generate a separate CONFIGURATION document and link from both?

Reply:
Hmm you're right! Maybe we can call it "Manual Configuration" instead of legacy since it's manual and it's more of a configuration process. We'll change the name of the file as well. 

@cjcenizal WDYT?

As for positioning, I believe we are positioning the quickstart model as the preferred model going forward. The manual method will still exist and we'll be able to use it. But placing our quickstart front and center:
- shows how easy it is to set up a Vectara-powered app, where arguably, very little doc-reading is needed to take the first step
- puts users in a context a bit more familiar to them (a workflow similar to create react app or npm init)

Having the documentation reflect that (where quickstart is up front, followed by manual configuration details) would only help to push this forward IMO, and get some buzz going about vectara-answer's incredible ease of use. Having both processes exist side-by-side without a clear preferred first option I'm concerned would muddle the experience of developers. 

To clarify- I think we should definitely have both configuration methods present and documented. IMO however we should keep the top-level readme as exists in the original PR branch, but add more detail via a manual config doc for those who'd rather use that. :)

Reply:
"Manual configuration" sounds good to me too. This direction makes sense to me.